src/allmydata/scripts/tahoe_manifest.py

```diff
@@ -96,8 +105,10 @@ def lineReceived(self, line):
                     if vc:
                         print(quote_output(vc, quotemarks=False), file=stdout)
                 else:
-                    print("%s %s" % (quote_output(d["cap"], quotemarks=False),
-                                               quote_path(d["path"], quotemarks=False)), file=stdout)
+                    # ensure_str() only necessary for Python 2.
+                    print(ensure_str("%s %s") % (
```

Comment:
Should the `ensure_str` actually wrap the post-interpolation string instead..?

Reply:
No, the issue is that `quote_output()` returns a native string, so you need the format string to also be native string, otherwise on Python 2 you get output that looks like `b'blah' b'foo'`.